mysql-test/suite/rocksdb/t/partition.test

```diff
@@ -658,6 +658,21 @@ INSERT INTO trb1 VALUES
 
 ALTER TABLE trb1 ADD PRIMARY KEY (id);
 
+--echo #
```

Comment:
Please make a common test include file shared between RocksDB and TokuDB (please check; there must be existing native partitioning examples).

Please then also start at 5.7, where only the testcase would be present.

Reply:
@laurynas-biveinis Done - #3132